### ISSUE TITLE
IOSXR: ShowL2VPNBridgeDomainDetail parser enhancement request

### ISSUE THREAD

**Issue by agandl046** [CiscoTestAutomation/genieparser#761]
Similar to the draft PR https://github.com/CiscoTestAutomation/genieparser/pull/744, I would like to get `Flow Label flags configured (Tx=1,Rx=1), negotiated (Tx=1,Rx=1)` parsed and added to the ShowL2VPNBridgeDomainDetail parser for Access PWs and VFIs. This is currently blocked by this bug in the parser: https://github.com/CiscoTestAutomation/genieparser/issues/760

Parser: https://github.com/CiscoTestAutomation/genieparser/blob/master/src/genie/libs/parser/iosxr/show_l2vpn.py#L1258

Example of output can be found in https://github.com/CiscoTestAutomation/genieparser/blob/master/src/genie/libs/parser/iosxr/tests/ShowL2vpnBridgeDomainDetail/cli/equal/golden7_output.txt

**Comment by iamsatyanarayan** [comment]
Hi @agandl046 
I will be working on your ticket. I'll get back to you once I have any update on it.

**Comment by iamsatyanarayan** [comment]
Hi @agandl046 
I am  working on this parser issue. I'll let you know  once I have any update on it.

**Comment by iamsatyanarayan** [comment]
Hi @agandl046 
PR has been raised for your issue.
PR link :- https://wwwin-github.cisco.com/pyATS/genieparser/pull/2834

**Comment by iamsatyanarayan** [comment]
Hi @agandl046   ,
Could you please provide any update on your issue ?

**Comment by agandl046** [comment]
Hi @iamsatyanarayan, 
I don't seem to have access to the PR you shared

**Comment by iamsatyanarayan** [comment]
Hi @agandlo46
PR has been merged. It will reflect in next release.

**Comment by agandl046** [comment]
@iamsatyanarayan Where can I see this update to confirm it works with my outputs? Also, this draft PR https://github.com/CiscoTestAutomation/genieparser/pull/744 is for a similar change. Is it still being worked on?

**Comment by iamsatyanarayan** [comment]
Hi @agandl046 
PR link :- https://wwwin-github.cisco.com/pyATS/genieparser/pull/2838

**Comment by agandl046** [comment]
@iamsatyanarayan Is that a Cisco internal github? That page doesn't load

**Comment by iamsatyanarayan** [comment]
Hi @agandl046 
Please wait until next release,

**Comment by iamsatyanarayan** [comment]
Hi @agandl046 
Could you please share with me the piestack ticket link for this PR :- https://github.com/CiscoTestAutomation/genieparser/pull/744

**Comment by agandl046** [comment]
Hi @iamsatyanarayan. I don't know if there was ever a ticket opened for that PR. I had originally reached out via email about that issue back in March and sent follow up emails in April/May/June with no response. The support email I had used was pyats-support-ext@cisco.com and Dave Wapstra was who originally responded.

**Comment by iamsatyanarayan** [comment]
HI,
Ok please give me some time . I will let you know after get confirmation from team.

**Comment by SohanTirpude** [comment]
Hello @agandl046,

The newer version(23.9) of pyATS has been released, hence kindly update your pyATS and try again. Kindly let me know the status.

**Comment by SohanTirpude** [comment]
Hello @agandl046,

Did you get a chance to test this fix? Kindly let me know.

**Comment by agandl046** [comment]
Hi @SohanTirpude. The fix looks good. Thank you!

**Comment by SohanTirpude** [comment]
Hello @agandl046, 

Thank you very much for the confirmation, hence I am closing this ticket. If you are facing any other issues, kindly open a new ticket. Thank you again.
